### ISSUE TITLE
Control plane load balancing on IPv6 does IPVS service error

### ISSUE THREAD

**Issue by RomLecat** [kube-vip/kube-vip#781]
Hi,

**Describe the bug**
I have the same error than #755 in IPv6, however the VIP is advertised as expected (ip a s dev eth0 shows the IPv6 VIP on one master node).
IPv4 VIP works without error.

Logs shows:

```
time="2024-03-07T18:53:02Z" level=info msg="IPVS Loadbalancer enabled for 1.2.1"
time="2024-03-07T18:53:02Z" level=info msg="Kube-Vip is watching nodes for control-plane labels"
time="2024-03-07T18:53:02Z" level=info msg="Gratuitous Arp broadcast will repeat every 3 seconds for [<vip>]"
time="2024-03-07T18:53:02Z" level=info msg="Broadcasting NDP update for <vip> (<mac>) via eth0"
time="2024-03-07T18:53:02Z" level=info msg="ndp: &{false false true <vip> [0xc000051c60]}"
time="2024-03-07T18:53:02Z" level=error msg="Error querying backends file does not exist"
time="2024-03-07T18:53:02Z" level=error msg="Unable to create an IPVS service, ensure IPVS kernel modules are loaded"
time="2024-03-07T18:53:02Z" level=fatal msg="IPVS service error: netlink receive: invalid argument"
```

**To Reproduce**
Helm values:

```yaml
kube-vip:
  image:
    tag: "v0.7.1"
  nameOverride: "kube-vip-v6"
  config:
    address: <vip>
  env:
    vip_arp: "true"
    vip_interface: eth0
    vip_port: "6443"
    lb_enable: "true"
    cp_enable: "true"
    svc_enable: "false"
    vip_cidr: "128"
    prometheus_server: ":2113"
  tolerations:
    - effect: NoSchedule
      key: node-role.kubernetes.io/master
    - effect: NoSchedule
      key: node-role.kubernetes.io/control-plane
      operator: Exists
    - effect: NoExecute
      key: node-role.kubernetes.io/master
  affinity:
    nodeAffinity:
      requiredDuringSchedulingIgnoredDuringExecution:
        nodeSelectorTerms:
          - matchExpressions:
            - key: node-role.kubernetes.io/master
              operator: Exists
          - matchExpressions:
            - key: node-role.kubernetes.io/control-plane
              operator: Exists
```

k3s is started with:

```
'--kube-proxy-arg=proxy-mode=ipvs' \
'--kube-proxy-arg=ipvs-strict-arp' \
'--kube-proxy-arg=ipvs-exclude-cidrs=<ipv4-net>/23,<ipv6-net>/64' \
```

**Expected behavior**
IPVS should not error.

**Environment (please complete the following information):**
 - OS/Distro: Debian 12
 - Kubernetes Version: v1.25.6+k3s1
 - Kube-vip Version: 0.7.1

Thanks !

**Comment by lubronzhan** [comment]
Interesting, what's the output of `ipvsadm -Ln` when ipv6 is enabled?
What's your node ip range?

**Comment by RomLecat** [comment]
Here you go :

```
root@k8s-master-1:~# ipvsadm -Ln
IP Virtual Server version 1.2.1 (size=4096)
Prot LocalAddress:Port Scheduler Flags
  -> RemoteAddress:Port           Forward Weight ActiveConn InActConn
TCP  <ipv4-public-prefix>.20:3129 rr
  -> 10.8.19.203:3129             Masq    1      0          0
TCP  <ipv4-public-prefix>.23:80 rr
  -> 10.8.3.90:80                 Masq    1      0          0
  -> 10.8.19.253:80               Masq    1      0          0
TCP  <ipv4-public-prefix>.23:443 rr
  -> 10.8.3.90:443                Masq    1      0          0
  -> 10.8.19.253:443              Masq    1      0          0
TCP  10.8.9.0:30151 rr
  -> 10.8.3.90:443                Masq    1      0          0
  -> 10.8.19.253:443              Masq    1      0          0
TCP  10.8.9.0:30394 rr
  -> 10.8.19.203:3129             Masq    1      0          0
TCP  10.8.9.0:31662 rr
  -> 10.8.3.199:80                Masq    1      0          0
  -> 10.8.19.9:80                 Masq    1      0          0
TCP  10.8.9.0:31851 rr
  -> 10.8.4.4:6980                Masq    1      0          0
TCP  10.8.9.0:31970 rr
  -> 10.8.3.90:80                 Masq    1      0          0
  -> 10.8.19.253:80               Masq    1      0          0
TCP  10.8.9.0:32524 rr
  -> 10.8.3.199:443               Masq    1      0          0
  -> 10.8.19.9:443                Masq    1      0          0
TCP  10.8.128.1:443 rr
  -> 10.9.2.241:6443              Masq    1      1          0
  -> 10.9.2.242:6443              Masq    1      0          0
  -> 10.9.2.243:6443              Masq    1      1          0
TCP  10.8.128.10:53 rr
  -> 10.8.4.13:53                 Masq    1      0          0
TCP  10.8.128.10:9153 rr
  -> 10.8.4.13:9153               Masq    1      0          0
TCP  10.8.132.200:9696 rr
  -> 10.8.3.235:9696              Masq    1      0          0
TCP  10.8.133.128:443 rr
  -> 10.8.4.70:5443               Masq    1      0          0
  -> 10.8.19.200:5443             Masq    1      0          0
TCP  10.8.133.183:5800 rr
  -> 10.8.19.203:5800             Masq    1      0          0
TCP  10.8.135.15:6767 rr
  -> 10.8.3.253:6767              Masq    1      0          0
TCP  10.8.138.21:8080 rr
  -> 10.8.19.60:8080              Masq    1      0          0
TCP  10.8.139.43:80 rr
  -> 10.8.3.212:80                Masq    1      0          0
TCP  10.8.145.129:8181 rr
  -> 10.8.19.164:8181             Masq    1      0          0
TCP  10.8.147.60:443 rr
  -> 10.8.19.62:10250             Masq    1      0          0
TCP  10.8.151.237:8600 rr
  -> 10.8.19.156:8600             Masq    1      0          0
TCP  10.8.153.60:4848 rr
  -> 10.8.3.167:4848              Masq    1      0          0
TCP  10.8.158.104:443 rr
  -> 10.8.4.55:9443               Masq    1      0          0
TCP  10.8.159.11:3003 rr
  -> 10.8.3.75:3003               Masq    1      0          0
TCP  10.8.159.18:8080 rr
  -> 10.8.3.42:8080               Masq    1      0          0
TCP  10.8.160.176:5432 rr
  -> 10.8.3.73:5432               Masq    1      0          0
TCP  10.8.160.188:443 rr
  -> 10.8.3.90:8443               Masq    1      0          0
  -> 10.8.19.253:8443             Masq    1      0          0
TCP  10.8.161.146:80 rr
  -> 10.8.4.47:80                 Masq    1      0          0
  -> 10.8.19.152:80               Masq    1      0          0
TCP  10.8.163.210:8096 rr
  -> 10.8.3.198:8096              Masq    1      0          0
TCP  10.8.164.78:7011 rr
  -> 10.8.4.154:7011              Masq    1      0          0
TCP  10.8.164.194:5473 rr
  -> 10.9.2.241:5473              Masq    1      0          0
  -> 10.9.2.242:5473              Masq    1      0          0
  -> 10.9.2.243:5473              Masq    1      0          0
TCP  10.8.165.165:8989 rr
  -> 10.8.3.88:8989               Masq    1      0          0
TCP  10.8.166.104:80 rr
  -> 10.8.19.183:8080             Masq    1      0          0
TCP  10.8.166.104:443 rr
  -> 10.8.19.183:8080             Masq    1      0          0
TCP  10.8.167.7:3129 rr
  -> 10.8.19.203:3129             Masq    1      0          0
TCP  10.8.169.41:6379 rr
  -> 10.8.19.251:6379             Masq    1      0          0
TCP  10.8.170.46:80 rr
  -> 10.8.3.90:80                 Masq    1      0          0
  -> 10.8.19.253:80               Masq    1      0          0
TCP  10.8.170.46:443 rr
  -> 10.8.3.90:443                Masq    1      0          0
  -> 10.8.19.253:443              Masq    1      0          0
TCP  10.8.173.224:7000 rr
  -> 10.8.4.135:7000              Masq    1      0          0
TCP  10.8.174.24:10050 rr
  -> 10.9.2.241:10050             Masq    1      0          0
  -> 10.9.2.242:10050             Masq    1      0          0
  -> 10.9.2.243:10050             Masq    1      0          0
  -> 10.9.2.246:10050             Masq    1      0          0
  -> 10.9.2.247:10050             Masq    1      0          0
  -> 10.9.2.248:10050             Masq    1      0          0
TCP  10.8.176.96:8080 rr
  -> 10.8.19.142:8080             Masq    1      0          0
TCP  10.8.178.75:8083 rr
  -> 10.8.3.105:8083              Masq    1      0          0
TCP  10.8.181.72:443 rr
  -> 10.8.3.199:8443              Masq    1      0          0
  -> 10.8.19.9:8443               Masq    1      0          0
TCP  10.8.182.57:9402 rr
  -> 10.8.4.164:9402              Masq    1      0          0
TCP  10.8.184.114:6980 rr
  -> 10.8.4.4:6980                Masq    1      0          0
TCP  10.8.191.122:7979 rr
  -> 10.8.3.246:7979              Masq    1      0          0
TCP  10.8.203.237:6379 rr
  -> 10.8.3.82:6379               Masq    1      0          0
TCP  10.8.204.7:5432 rr
  -> 10.8.19.198:5432             Masq    1      0          0
TCP  10.8.214.216:5432 rr
  -> 10.8.3.183:5432              Masq    1      0          0
TCP  10.8.221.153:8081 rr
  -> 10.8.4.166:8081              Masq    1      0          0
TCP  10.8.222.3:9000 rr
  -> 10.8.4.120:9000              Masq    1      0          0
TCP  10.8.222.21:3001 rr
  -> 10.8.3.166:3001              Masq    1      0          0
TCP  10.8.225.147:6666 rr
  -> 10.8.3.24:6666               Masq    1      0          0
TCP  10.8.229.253:8080 rr
  -> 10.8.4.18:8080               Masq    1      0          0
TCP  10.8.238.187:80 rr
  -> 10.8.3.199:80                Masq    1      0          0
  -> 10.8.19.9:80                 Masq    1      0          0
TCP  10.8.238.187:443 rr
  -> 10.8.3.199:443               Masq    1      0          0
  -> 10.8.19.9:443                Masq    1      0          0
TCP  10.8.240.108:7878 rr
  -> 10.8.3.14:7878               Masq    1      0          0
TCP  10.8.241.20:8191 rr
  -> 10.8.3.46:8191               Masq    1      0          0
TCP  10.8.246.95:443 rr
  -> 10.8.19.241:10250            Masq    1      0          0
TCP  10.8.247.59:5556 rr
  -> 10.8.19.28:5556              Masq    1      0          0
TCP  10.8.247.59:5557 rr
  -> 10.8.19.28:5557              Masq    1      0          0
TCP  10.9.1.111:6980 rr
  -> 10.8.4.4:6980                Masq    1      0          0
TCP  10.9.1.112:80 rr
  -> 10.8.3.199:80                Masq    1      0          0
  -> 10.8.19.9:80                 Masq    1      0          0
TCP  10.9.1.112:443 rr
  -> 10.8.3.199:443               Masq    1      0          0
  -> 10.8.19.9:443                Masq    1      0          0
TCP  10.9.2.241:30151 rr
  -> 10.8.3.90:443                Masq    1      0          0
  -> 10.8.19.253:443              Masq    1      0          0
TCP  10.9.2.241:30394 rr
  -> 10.8.19.203:3129             Masq    1      0          0
TCP  10.9.2.241:31662 rr
  -> 10.8.3.199:80                Masq    1      0          0
  -> 10.8.19.9:80                 Masq    1      0          0
TCP  10.9.2.241:31851 rr
  -> 10.8.4.4:6980                Masq    1      0          0
TCP  10.9.2.241:31970 rr
  -> 10.8.3.90:80                 Masq    1      0          0
  -> 10.8.19.253:80               Masq    1      0          0
TCP  10.9.2.241:32524 rr
  -> 10.8.3.199:443               Masq    1      0          0
  -> 10.8.19.9:443                Masq    1      0          0
TCP  10.9.2.254:6443 rr
  -> 10.9.2.241:6443              Local   1      0          0
  -> 10.9.2.242:6443              Local   1      0          0
  -> 10.9.2.243:6443              Local   1      0          0
TCP  10.9.2.254:30151 rr
  -> 10.8.3.90:443                Masq    1      0          0
  -> 10.8.19.253:443              Masq    1      0          0
TCP  10.9.2.254:30394 rr
  -> 10.8.19.203:3129             Masq    1      0          0
TCP  10.9.2.254:31662 rr
  -> 10.8.3.199:80                Masq    1      0          0
  -> 10.8.19.9:80                 Masq    1      0          0
TCP  10.9.2.254:31851 rr
  -> 10.8.4.4:6980                Masq    1      0          0
TCP  10.9.2.254:31970 rr
  -> 10.8.3.90:80                 Masq    1      0          0
  -> 10.8.19.253:80               Masq    1      0          0
TCP  10.9.2.254:32524 rr
  -> 10.8.3.199:443               Masq    1      0          0
  -> 10.8.19.9:443                Masq    1      0          0
UDP  10.8.128.10:53 rr
  -> 10.8.4.13:53                 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:700::5aa6:4240]:30151 rr
  -> [<ipv6-public-prefix-1>:700::66d:75b7]:443 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4f1]:443 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:700::5aa6:4240]:30394 rr
  -> [<ipv6-public-prefix-1>:700::44c3:c4f7]:3129 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:700::5aa6:4240]:31662 rr
  -> [<ipv6-public-prefix-1>:700::66d:75af]:80 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4ca]:80 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:700::5aa6:4240]:31851 rr
  -> [<ipv6-public-prefix-1>:700::9c31:c102]:6980 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:700::5aa6:4240]:31970 rr
  -> [<ipv6-public-prefix-1>:700::66d:75b7]:80 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4f1]:80 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:700::5aa6:4240]:32524 rr
  -> [<ipv6-public-prefix-1>:700::66d:75af]:443 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4ca]:443 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:701::11d9]:6980 rr
  -> [<ipv6-public-prefix-1>:700::9c31:c102]:6980 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:701::3d82]:80 rr
  -> [<ipv6-public-prefix-1>:700::66d:75b7]:80 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4f1]:80 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:701::3d82]:443 rr
  -> [<ipv6-public-prefix-1>:700::66d:75b7]:443 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4f1]:443 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:701::8ee0]:80 rr
  -> [<ipv6-public-prefix-1>:700::66d:75af]:80 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4ca]:80 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:701::8ee0]:443 rr
  -> [<ipv6-public-prefix-1>:700::66d:75af]:443 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4ca]:443 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:701::d91b]:3129 rr
  -> [<ipv6-public-prefix-1>:700::44c3:c4f7]:3129 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:811::cafe:0]:3129 rr
  -> [<ipv6-public-prefix-1>:700::44c3:c4f7]:3129 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:811::cafe:3]:80 rr
  -> [<ipv6-public-prefix-1>:700::66d:75b7]:80 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4f1]:80 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:811::cafe:3]:443 rr
  -> [<ipv6-public-prefix-1>:700::66d:75b7]:443 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4f1]:443 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:916::cafe:0]:6980 rr
  -> [<ipv6-public-prefix-1>:700::9c31:c102]:6980 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:916::cafe:1]:80 rr
  -> [<ipv6-public-prefix-1>:700::66d:75af]:80 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4ca]:80 Masq    1      0          0
TCP  [<ipv6-public-prefix-1>:916::cafe:1]:443 rr
  -> [<ipv6-public-prefix-1>:700::66d:75af]:443 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4ca]:443 Masq    1      0          0
TCP  [<ipv6-public-prefix-2>:2::b002:1]:30151 rr
  -> [<ipv6-public-prefix-1>:700::66d:75b7]:443 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4f1]:443 Masq    1      0          0
TCP  [<ipv6-public-prefix-2>:2::b002:1]:30394 rr
  -> [<ipv6-public-prefix-1>:700::44c3:c4f7]:3129 Masq    1      0          0
TCP  [<ipv6-public-prefix-2>:2::b002:1]:31662 rr
  -> [<ipv6-public-prefix-1>:700::66d:75af]:80 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4ca]:80 Masq    1      0          0
TCP  [<ipv6-public-prefix-2>:2::b002:1]:31851 rr
  -> [<ipv6-public-prefix-1>:700::9c31:c102]:6980 Masq    1      0          0
TCP  [<ipv6-public-prefix-2>:2::b002:1]:31970 rr
  -> [<ipv6-public-prefix-1>:700::66d:75b7]:80 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4f1]:80 Masq    1      0          0
TCP  [<ipv6-public-prefix-2>:2::b002:1]:32524 rr
  -> [<ipv6-public-prefix-1>:700::66d:75af]:443 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4ca]:443 Masq    1      0          0
TCP  [<ipv6-public-prefix-2>:2::b002:ffff]:30151 rr
  -> [<ipv6-public-prefix-1>:700::66d:75b7]:443 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4f1]:443 Masq    1      0          0
TCP  [<ipv6-public-prefix-2>:2::b002:ffff]:30394 rr
  -> [<ipv6-public-prefix-1>:700::44c3:c4f7]:3129 Masq    1      0          0
TCP  [<ipv6-public-prefix-2>:2::b002:ffff]:31662 rr
  -> [<ipv6-public-prefix-1>:700::66d:75af]:80 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4ca]:80 Masq    1      0          0
TCP  [<ipv6-public-prefix-2>:2::b002:ffff]:31851 rr
  -> [<ipv6-public-prefix-1>:700::9c31:c102]:6980 Masq    1      0          0
TCP  [<ipv6-public-prefix-2>:2::b002:ffff]:31970 rr
  -> [<ipv6-public-prefix-1>:700::66d:75b7]:80 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4f1]:80 Masq    1      0          0
TCP  [<ipv6-public-prefix-2>:2::b002:ffff]:32524 rr
  -> [<ipv6-public-prefix-1>:700::66d:75af]:443 Masq    1      0          0
  -> [<ipv6-public-prefix-1>:700::44c3:c4ca]:443 Masq    1      0          0
 ```

master-1 (current master) has <ipv6-public-prefix-2>:2::b002:1/64 IP and <ipv6-public-prefix-2>:2::b002:ffff/128 VIP.

**Comment by lubronzhan** [comment]
Thanks. just noticed that our logger doesn't print file name and line, that's something we should change.

So the error you showed `IPVS service error: netlink receive: invalid argument` comes from here  https://github.com/kube-vip/kube-vip/blob/main/pkg/loadbalancer/ipvs.go#L126-L128. Looks like the called library does support ipv6, it's just maybe the configuration for service is incorrect.

So the PR for the [issue](https://github.com/kube-vip/kube-vip/issues/755) you mentioned https://github.com/kube-vip/kube-vip/commit/388b6ca27f396d78ba62b3eaac25315c0af1440f#diff-cbd55604ef85d846c166257530d75148facedb500234aee4e09c6cbf7d8004adR61, does add the netmask, but that netmask is only for ipv4 case, should check if ipfamily is ipv6, use different netmask, I can create a PR to fix it.

**Comment by lubronzhan** [comment]
Once the PR is merged would you mind creating a dev image and test it out? Since I don't have a live env to debug

**Comment by RomLecat** [comment]
The error is now gone with the fix (I built an image, available in ghcr.io/romlecat/kube-vip:0.7.1-fix781 if anyone is interrested), it looks better:

```
time="2024-03-10T11:10:24Z" level=error msg="Error querying backends file does not exist"
time="2024-03-10T11:10:24Z" level=warning msg="load balancer for API server already exists, attempting to remove and re-create"
time="2024-03-10T11:10:24Z" level=info msg="Created Load-Balancer services on [<ipv6-public-prefix-2>:2::b002:ffff:6443]"
time="2024-03-10T11:10:24Z" level=info msg="Broadcasting NDP update for <ipv6-public-prefix-2>:2::b002:ffff (bc:24:11:5b:e0:8d) via eth0"
time="2024-03-10T11:10:24Z" level=info msg="ndp: &{false false true <ipv6-public-prefix-2>:2::b002:ffff [0xc0006eea00]}"
time="2024-03-10T11:10:27Z" level=info msg="Broadcasting NDP update for <ipv6-public-prefix-2>:2::b002:ffff (bc:24:11:5b:e0:8d) via eth0"
time="2024-03-10T11:10:27Z" level=info msg="ndp: &{false false true <ipv6-public-prefix-2>:2::b002:ffff [0xc000051100]}"
time="2024-03-10T11:10:30Z" level=info msg="Broadcasting NDP update for <ipv6-public-prefix-2>:2::b002:ffff (bc:24:11:5b:e0:8d) via eth0"
time="2024-03-10T11:10:30Z" level=info msg="ndp: &{false false true <ipv6-public-prefix-2>:2::b002:ffff [0xc000051da0]}"
time="2024-03-10T11:10:33Z" level=info msg="Broadcasting NDP update for <ipv6-public-prefix-2>:2::b002:ffff (bc:24:11:5b:e0:8d) via eth0"
time="2024-03-10T11:10:33Z" level=info msg="ndp: &{false false true <ipv6-public-prefix-2>:2::b002:ffff [0xc0002a4bc0]}"
time="2024-03-10T11:10:34Z" level=error msg="Error querying backends file does not exist"
time="2024-03-10T11:10:34Z" level=warning msg="load balancer for API server already exists, attempting to remove and re-create"
time="2024-03-10T11:10:34Z" level=info msg="Created Load-Balancer services on [<ipv6-public-prefix-2>:2::b002:ffff:6443]"
time="2024-03-10T11:10:34Z" level=error msg="Error querying backends file does not exist"
time="2024-03-10T11:10:34Z" level=warning msg="load balancer for API server already exists, attempting to remove and re-create"
time="2024-03-10T11:10:34Z" level=info msg="Created Load-Balancer services on [<ipv6-public-prefix-2>:2::b002:ffff:6443]"
time="2024-03-10T11:10:34Z" level=error msg="Error querying backends file does not exist"
time="2024-03-10T11:10:34Z" level=warning msg="load balancer for API server already exists, attempting to remove and re-create"
time="2024-03-10T11:10:34Z" level=info msg="Created Load-Balancer services on [<ipv6-public-prefix-2>:2::b002:ffff:6443]"
```

The IPv4 and IPv6 instances seems to conflict, is there anyway to fix this ?

**Comment by lubronzhan** [comment]
Hi @RomLecat could you create a new issue about above error, I can try look into the new issue. Not sure if it's conflict or is it because failing to find existing rule

**Comment by RomLecat** [comment]
Sure, thanks! Here it is: #784
